tech.hyperjump.monika.yml

```diff
@@ -0,0 +1,27 @@
+id: tech.hyperjump.monika
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: monika
+finish-args:
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: monika
+    buildsystem: simple
+    build-options:
+      env:
+        npm_config_offline: 'true'
+    build-commands:
+      - unzip monika*.zip
+      - install -D -t /app/bin monika
+      - install -D -t /app/bin node_sqlite3.node
+    sources:
+      - type: file
+        url: https://github.com/hyperjumptech/monika/releases/download/v1.6.13/monika-v1.6.13-linux-x64.zip
```

Comment:
so you need to tell flathub to not build `aarch64`, or build from source so that this is available on `aarch64`